packages/ui/src/components/va-carousel/VaCarousel.stories.ts

```diff
@@ -1,12 +1,224 @@
-import { defineComponent } from 'vue'
-import VaCarousel from './VaCarousel.demo.vue'
+import { userEvent } from '../../../.storybook/interaction-utils/userEvent'
+import { within } from '@storybook/testing-library'
+import { VaCarousel } from '../va-carousel'
+import { expect } from '@storybook/jest'
+import { sleep } from '../../utils/sleep'
+
+const getItems = () => [
+  'https://picsum.photos/id/10/2500',
+  'https://picsum.photos/id/14/2500',
+  'https://picsum.photos/id/18/2500',
+]
 
 export default {
   title: 'VaCarousel',
   component: VaCarousel,
+  tags: ['autodocs'],
+}
+
+export const Default = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems(), current: 0 }),
+  template: `
+    <VaCarousel :items="items" />
+  `,
+})
+
+export const Vertical = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems() }),
+  template: `
+    <VaCarousel :items="items" vertical />
+  `,
+})
+
+export const Arrows = () => ({
+  components: { VaCarousel },
+  data: () => ({ items: getItems(), current: 0 }),
+  template: `
+    [true]
+    <VaCarousel :items="items" :indicators="false" v-model="current"/>
+    [false]
+    <VaCarousel :items="items" :indicators="false" :arrows="false" v-model="current"/>
+  `,
+})
+
+Arrows.play = async ({ canvasElement, step }) => {
+  const canvas = within(canvasElement)
+  const slides = document.querySelectorAll('div.va-carousel__slide')
+  const NavigatesOnArrowClick = canvas.getByRole('button', { name: 'go next slide' }) as HTMLElement
+
+  await step('Should head to slide 2 by clicking on arrow', async () => {
```

Comment:
Can we use test step naming like this? - "Changes slide on arrow click".

When test breaks it would be easier to understand what was broken. (as very likely if it doesn't navigate to slide 2 - then it doesn't navigate at all!)

Same idea apply for all other stories.